openedx/features/survey_report/admin.py

```diff
@@ -0,0 +1,89 @@
+"""
+Django Admin page for SurveyReport.
+"""
+
+
+from django.contrib import admin
+from django import forms
+from .models import SurveyReport
+
+
+class SurveyReportForm(forms.ModelForm):
+    """
+    ModelForm to survey report.
+    """
+    class Meta:
+        model = SurveyReport
+        help_texts = {
+            'courses_offered': 'Total number of active unique courses.',
```

Comment:
Is there a reason you favor putting this here instead of the model.py file?
If there is a compeling reason, this would be fine, but otherwise I would suggest we put them at the place the field is defined.

Reply:
@felipemontoya Now I've remember the reason, because if I add the help_text to the model, I need to create a new Migration.

Reply:
Thanks @Alec4r. I still think it is better this way.